Bottom: 6092dfe2e626693b91c2c1debad939111aa2169b
Top:    0010e1c997fe76482c35cf0483a5cf39cad71d10
Author: scott snyder <sss@karma>
Date:   2020-02-03 00:31:45 -0500

Refresh of cmake

---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 556667f..648da9a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -232,5 +232,6 @@ DISPLAY_STD_VARIABLES()
 
 
 # generate and install following configuration files
-GENERATE_PACKAGE_CONFIGURATION_FILES( CondDBMySQLConfig.cmake CondDBMySQLConfigVersion.cmake CondDBMySQLLibDeps.cmake )
+GENERATE_PACKAGE_CONFIGURATION_FILES( CondDBMySQLConfig.cmake CondDBMySQLConfigVersion.cmake  )
 
+INSTALL (FILES ${PROJECT_SOURCE_DIR}/cmake/CondDBMySQLLibDeps.cmake DESTINATION lib/cmake )
diff --git a/cmake/CondDBMySQLLibDeps.cmake b/cmake/CondDBMySQLLibDeps.cmake
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/cmake/CondDBMySQLLibDeps.cmake
@@ -0,0 +1 @@
+
